RC 82: Manuscript Cases and other changes (#87)
highfidelity/hifi-docs-sphinx: docs/source/explore/get-started/vr-controls.rst

> 

Reviewer: Since we're focusing less on HMD and more on Desktop, we should remove any reference to "VR equipment is better than Desktop". So change the first paragraph to something more vague like 

```suggestion
To interact with people in 3D and track body movements, you will need to use VR equipment such as the Oculus Rift or HTC Vive. With VR equipment, you can have a fully immersive experience in the 3D metaverse.
```

Author: Should we use the word metaverse? I feel like we're shifting away from what it means to a lot of users. Can we replace that with just High Fidelity instead?

Author: I modified this line and made it High Fidelity instead.

> .. note:: If you want to jump and fly in VR, you must enable it under Settings > Controls > VR Movement.

Reviewer: ```suggestion
.. note:: If you want to jump, fly, or strafe (walk sideways) in VR, you must enable it under Settings > Controls > VR Movement.
```

| Hand-Relative              |                                                                                 |
+----------------------------+---------------------------------------------------------------------------------+
| *Movement Direction:*      | When selected, your avatar will move in the direction your hand is pointing,    |
| Hand-Relative (Level)      | without taking pitch into account.                                              |
+----------------------------+---------------------------------------------------------------------------------+
| Dominant Hand              | Select 'Left' or 'Right'. Teleport and turning controls move to the controller  |
|                            | in the dominant hand.                                                           |
+----------------------------+---------------------------------------------------------------------------------+
| *Control Scheme Selection:*| In Default mode, your walking speed doesn't change depending on how far forward |
| Default                    | you push your controller's joystick. Fully pushing your joystick forward will   |

> |                            | make your avatar run at twice your walking speed.                               |

Reviewer: ```suggestion
|                            | make your avatar run.                               |
```

I don't think its important to mention that running is twice your walking speed. I think most people think of run as "faster than walking" so its needless to say what the technical detail of that is. Plus, once you get into Analog and Analog++, what does twice the walking speed mean since your walkign speed varies based on how far forward you push the joystick. So we'll stick with "walk or run".

+----------------------------+---------------------------------------------------------------------------------+
| *Control Scheme Selection:*| In Analog mode, your walking speed changes based on how far forward you push    |
| Analog                     | your controller's joystick. Fully pushing your joystick forward will            |

> |                            | make your avatar run at twice your walking speed.                               |

Reviewer: ```suggestion
|                            | make your avatar run.                           |
```

+----------------------------+---------------------------------------------------------------------------------+
| *Control Scheme Selection:*| In Analog++ mode, your walking speed changes based on how far forward you push  |
| Analog++                   | your controller's joystick. You can also use the slider to change the walking   |
|                            | speed in meters/second. Fully pushing your joystick forward will make your      |

> |                            | avatar run at twice your walking speed.                                         |

Reviewer: ```suggestion
|                            | avatar run.                                      |
```

+----------------------------+---------------------------------------------------------------------------------+

-------------------------

> Gamepad

Reviewer: Does Gamepad belong in the Desktop page? I feel like its not likely to be used with an HMD, but rather with a computer...

Author: This seems to be something no one is really agreeing on - gamepads are also used for VR experiences. I'm going to leave it as it is and create a manuscript case for this.

-------------------------

> While the use of a gamepad is supported, High Fidelity is best experienced with hand controllers like the Oculus Touch, HTC Vive, and Microsoft MR controllers.

Reviewer: ```suggestion
While the use of a gamepad is supported, High Fidelity is best experienced with VR equipment or the keyboard in Desktop mode.
```
